base_layer/core/src/transactions/coinbase_builder.rs

```diff
@@ -21,15 +21,13 @@
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
-use chacha20poly1305::aead::OsRng;
 use log::*;
 use tari_common_types::{
     tari_address::TariAddress,
-    types::{Commitment, PrivateKey, PublicKey},
+    types::{Commitment, PrivateKey},
 };
-use tari_crypto::keys::PublicKey as PK;
 use tari_key_manager::key_manager_service::{KeyManagerInterface, KeyManagerServiceError};
-use tari_script::{one_sided_payment_script, stealth_payment_script, ExecutionStack, TariScript};
+use tari_script::{push_pubkey_script, ExecutionStack, TariScript};
```

Comment:
I specifically changed it away from that as I want to avoid confusion in the code base between the two. The same script is used for interactive, one-sided and one-sided stealth